src/main/kotlin/com/workos/mfa/models/Challenge.kt

```diff
@@ -34,5 +34,5 @@ data class Challenge
 
   @JvmField
   @JsonProperty("authentication_factor_id")
-  val authenticationFactorId: String,
+  val authenticationauthenticationFactorId: String,
```

Comment:
Looks like there's a type here:

```suggestion
  val authenticationFactorId: String,
```